normandy/recipes/tests/actions/show-heartbeat.js

```diff
@@ -186,11 +184,9 @@ describe('ShowHeartbeatAction', function() {
 
         await action.execute();
         expect(this.normandy.showHeartbeat).toHaveBeenCalledWith(jasmine.objectContaining({
-            extraTelemetryArgs: {
-                surveyId: 'my-survey',
-                surveyVersion: 42,
-                testing: 1,
-            },
+            surveyId: 'my-survey',
+            surveyVersion: 42,
```

Comment:
You can leave out `surveyId` and `surveyVersion` here since this test is only about checking if the `testing` argument is present. The others are handled in a different test.